Automattic/jetpack
# Social: Add Nextdoor connection toggle

This makes the connections API call return Nextdoor connections and ensures that the icon has the correct colour.

## Proposed changes:

This adds Nextdoor to the 'known' connections when retrieving them via the API calls. It also adds the default colour to the icon for when it's displayed in the editor.

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:

* Create a Nextdoor connection on your site using D125526-code
* Load the editor and check that you can see the Nextdoor logo representing the connection

<img width="256" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/5d3db448-981a-415c-aea7-5f74dffd35a4">
